client/src/components/CinemaHall/index.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-plusplus */
+import React, { useContext } from 'react';
+import Seat from '../Seat';
+import './style.scss';
+import ReservationProvider from '../../context/Reservation';
+
+const CinemaHall = () => {
+  const nrOfRows = 7;
+  const nrOfColumns = 8;
+  const columns = [];
+  const rowsInLetter = [];
+
+  for (let i = 1; i <= nrOfColumns; i++) {
+    columns.push(i);
+  }
+  for (let j = 1; j <= nrOfRows; j++) {
+    rowsInLetter.push(String.fromCharCode(j + 64));
+  }
+
+  return (
+    <ReservationProvider>
+      <div>
+        <div className="cinema__hall__screen">Screen this way!</div>
+        <ul>
+          {rowsInLetter.map((row) => {
+            return (
+              <li className="cinema__hall__row" key={row}>
+                <div>{row}</div>
```

Comment:
Use <p> tag in cases like this. It is a simple text, doesn't need to be a <div>.